test/E2ETests/E2EApps/E2EApp/E2EApp.csproj

```diff
@@ -42,8 +42,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.WebUtilities" Version="2.2.0" />
-    <PackageReference Condition="$(TestBuild) == 'true'" Include="Microsoft.Azure.Functions.Worker" Version="1.10.0" />
```

Comment:
@satvu at the top of this file we have checks for the `FunctionsRuntimeVersion` to determine the target framework. I'm a little confused on why we check if the runtime version is 4 for us to use .net 7? 

Same at the bottom of the file with `System.Text.Json`

Reply:
When we upgraded this app to V4, it began failing the V3 integration tests (https://github.com/Azure/azure-functions-dotnet-worker/issues/1060).

https://github.com/Azure/azure-functions-dotnet-worker/pull/1147 - the PR to resolve that was a while back, but we needed to adjust for integration tests by allowing both V3 and V4 tests to run. From a design meeting it was decided this is the quickest/shortest fix. 

Other language workers (or so I'm told) have setups to test V3/V4 (like different branches/builds) but we just have this one app that runs in the integration tests.

Reply:
Ahh gotcha! So we don't run v3 tests anymore now that it is out of service - do we still need this in place?

Reply:
I think we should be ok to remove it?

cc: @Francisco-Gamino - do you still need this for integration testing?

Reply:
Yes, we do. Even though Functions v3 has reached the EOL, we need to have the ability to validate a v3 build in case we need to do another release.

/cc @fabiocav

Reply:
Good to know, thank you!